src/Standards/Squiz/Sniffs/WhiteSpace/MemberVarSpacingSniff.php

```diff
@@ -45,23 +45,30 @@ protected function processMemberVar(File $phpcsFile, $stackPtr)
     {
         $tokens = $phpcsFile->getTokens();
 
+        $stopPoints = [
+            T_SEMICOLON,
+            T_OPEN_CURLY_BRACKET,
+            T_CLOSE_CURLY_BRACKET,
+            T_OPEN_TAG,
```

Comment:
Would it be a good idea to include a test case that has `T_OPEN_TAG` as the end of the previous statement?

Reply:
Valid question. I kind of only added that token for redundancy, as using it would result in a parse error: https://3v4l.org/otur9

I suppose I could add the code snippet I used in 3v4l as a test, but not sure if it actually adds value.

Reply:
Considering it's a parse error - I could just remove the `T_OPEN_TAG` I suppose...

Reply:
Thanks :shipit: